test/integration/targets/vyos_lldp_global/tests/cli/replaced.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- debug:
+      msg: "START vyos_lldp_global replaced integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate.yaml
+
+- block:
+  - name: Replace device configurations of LLDP service with provided configurations
+    vyos_lldp_global: &replaced
+        config:
+            legacy_protocols:
+                - 'edp'
+                - 'sonmp'
+                - 'cdp'
+            address: 198.50.100.55
```

Comment:
IP addresses in integration tests should be in documentation range. Please refer to https://tools.ietf.org/html/rfc5737.

Reply:
Done